### PR TITLE
Improve sbt-test/http-backend/netty-channel-options

### PR DESCRIPTION
**Pull request by dwijnand**

### CODE REVIEW

File: dev-mode/sbt-plugin/src/main/scala/play/sbt/run/PlayRun.scala [hunk 1]
```diff
@@ -282,24 +282,19 @@ object PlayRun extends PlayRunCompat {
...
+  def stop(stagingDirectory: File): Unit = {
+    val pidFile = stagingDirectory / "RUNNING_PID"
+    if (pidFile.exists) {
       val pid = IO.read(pidFile)
       kill(pid)
-      // PID file will be deleted by a shutdown hook attached on start in ServerStart.scala
+      // PID file will be deleted by a shutdown hook attached on start in ProdServerStart.scala
       println(s"Stopped application with process ID $pid")
-    }
+    } else println(s"No PID file found at $pidFile. Are you sure the app is running?")
```
Comment (marcospereira) [inline dev-mode/sbt-plugin/src/main/scala/play/sbt/run/PlayRun.scala]: 👍

File: dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
-    InputKey[Unit]("callIndex") := {
-      DevModeBuild.callIndex()
-    },
-    InputKey[Unit]("checkLines") := {
-      val args                  = Def.spaceDelimited("<source> <target>").parsed
-      val source :: target :: _ = args
-      DevModeBuild.checkLines(source, target)
-    }
-  )
+               updateOptions := updateOptions.value.withLatestSnapshots(false)
```
Comment (marcospereira) [inline dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt]: For the sake of keeping things even cleaner, we can also add:

```scala
evictionWarningOptions in update := EvictionWarningOptions.default
    .withWarnTransitiveEvictions(false)
    .withWarnDirectEvictions(false)
```

File: dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt [hunk 3]
```diff
@@ -1,27 +1,33 @@
...
+
 organization := "com.lightbend.play"
+        name := "netty-channel-options"
+     version := "1.0-SNAPSHOT"
 
-version := "1.0-SNAPSHOT"
+enablePlugins(PlayScala, PlayNettyServer)
+disablePlugins(PlayAkkaHttpServer)
 
-lazy val root = (project in file("."))
```
Comment (marcospereira) [inline dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt]: Any specific reason to use a bare/flat definition here? I don't see a problem in terms of style, but I would prefer to keep things consistent with the other tests.

File: dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/project/Build.scala [hunk 4]
```diff
@@ -21,37 +17,24 @@ object DevModeBuild {
...
-    (status, contents)
+    conn.setConnectTimeout(10000)
+    conn.setReadTimeout(10000)
+    headers.foreach { case (k, v) => conn.setRequestProperty(k, v) }
+    try {
+      val status   = conn.getResponseCode
+      val in       = if (status < 400) conn.getInputStream else conn.getErrorStream
+      val contents = if (in == null) "" else try IO.readStream(in) finally in.close()
+      (status, contents)
+    } finally conn.disconnect()
```
Comment (marcospereira) [inline dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/project/Build.scala]: 👍

File: dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt (hunk 3 above)
Comment (dwijnand) [reply dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt]: Less boilerplate, less noise.

Comment (dwijnand) [reply dev-mode/sbt-plugin/src/sbt-test/http-backend/netty-channel-options/build.sbt]: I agree with the desire for _eventual_ consistency 😉

**Comment by dwijnand** [comment]
Superseded by #9575 #9576 #9577 #9357.

I might still recover https://github.com/playframework/playframework/pull/9571/commits/82af27a4b1169db3b90b70eb9487a127d7f3b316

**Pull request closed without merging**